clients/fides-js/src/lib/ui-utils.ts

```diff
@@ -0,0 +1,6 @@
+export const blockPageScrolling = () => {
+  document.body.style.overflowY = "hidden";
```

Comment:
it feels a bit dangerous to be setting styles this way. Another option would be to add a class to `body` and then update the styles accordingly in `fides-js/src/components/fides.css`. I'm unsure if one approach is necessarily better than the other, but with setting a class, we can at least have more control over any CSS overrides / changes the customer wishes to implement manually, right?

My main concern is because fides.js loads on so many browsers across the internet, I'm unsure how this might interact with other sites' specific functionality, so I at least want to keep an eye on it as we roll this out to customers.

Reply:
Yes, I think the class would be better. I actually coppied this approach from what I found on a11y-dialog.tsx. Let me see if I can change it quickly.